### PR TITLE
Hotfix/in app notification

### PR DESCRIPTION
**Pull request by ALVINdimpos**
#### What does this PR do?
  Fix in app notification 
#### Description of Task to be completed?
 Fix in app notification 
#### How should this be manually tested?

```
git clone https://github.com/atlp-rwanda/ecommerce-app-predators-bn.git
cd ecommerce-app-predators-bn
git checkout branch-hotfix/in-app-notification
npm i
npm run dev
```

#### What are the relevant pivotal tracker/Trello stories?

Trello [Link]() (#-?)

### CODE REVIEW

**Review by faid-terence — APPROVED** [review]

**Review by yobuir — APPROVED** [review]

**Review by bryan936M — APPROVED** [review]

File: .nyc_output/processinfo/dce50d59-aae9-4c87-9b28-1efe2de63e38.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"parent":null,"pid":6879,"argv":["/usr/local/bin/node","/Users/andelarwanda/Desktop/ecommerce-app-predators-bn/node_modules/.bin/mocha","--config","./tests/.mocharc.cjs"],"execArgv":[],"cwd":"/Users/andelarwanda/Desktop/ecommerce-app-predators-bn","time":1686075034595,"ppid":6873,"coverageFilename":"/Users/andelarwanda/Desktop/ecommerce-app-predators-bn/.nyc_output/dce50d59-aae9-4c87-9b28-1efe2de63e38.json","externalId":"","uuid":"dce50d59-aae9-4c87-9b28-1efe2de63e38","files":[]}
```
Comment (chanceDira) [inline .nyc_output/processinfo/dce50d59-aae9-4c87-9b28-1efe2de63e38.json]: This file can be in .gitignore

**Review by chanceDira — CHANGES_REQUESTED** [review]
Nice work, just one thing

File: .nyc_output/processinfo/dce50d59-aae9-4c87-9b28-1efe2de63e38.json (hunk 1 above)
Comment (ALVINdimpos) [reply .nyc_output/processinfo/dce50d59-aae9-4c87-9b28-1efe2de63e38.json]: solved!

Comment (chanceDira) [reply .nyc_output/processinfo/dce50d59-aae9-4c87-9b28-1efe2de63e38.json]: Cool.

**Review by chanceDira — APPROVED** [review]

**Pull request merged**